component/layout.js

```diff
@@ -0,0 +1,15 @@
+import { useRouter } from "next/router";
```

Comment:
Lets make this a typescript file, and also can we put this into a folder called layouts inside the components folder?